lib/driverProviders/driverProvider.ts

```diff
@@ -93,56 +90,44 @@ export abstract class DriverProvider {
    *
    * @param drivers {webdriver.WebDriver[]} The webdriver instances
    */
-  static quitDrivers(provider: DriverProvider, drivers: WebDriver[]): q.Promise<void> {
-    let deferred = q.defer<void>();
-    wdpromise
-        .all(drivers.map((driver: WebDriver) => {
-          return provider.quitDriver(driver);
-        }))
-        .then(
-            () => {
-              deferred.resolve();
-            },
-            () => {
-              deferred.resolve();
-            });
-    return deferred.promise;
+  static async quitDrivers(provider: DriverProvider, drivers: WebDriver[]): Promise<void> {
+    await Promise.all(drivers.map(async (driver: WebDriver) => {
```

Comment:
Awaiting in the map will serialize all the promises. It's not super important, but you could just do
```
await Promise.all(drivers.map((driver: WebDriver) => {
     return provider.quitDriver(driver);
}));
```
which would actually wait for all the promises to resolve in parallel.

Reply:
Oh yup, that's right. Fixing.